DuckDuckGo/TabViewController.swift

```diff
@@ -71,6 +71,10 @@ class TabViewController: UIViewController {
     private var lastError: Error?
     private var shouldReloadOnError = false
     private var failingUrls = Set<String>()
+    
+    private var pageNetworkNames = Set<String>()
```

Comment:
This is basically a tracker's network name right? So maybe let's change the name to something that resembles that, as I initially thought that 'pageNetworkNames' refers to visited page(s).